tests/test_chat.py

```diff
@@ -28,8 +28,8 @@
 def chat(bot):
     return Chat(
         TestChat.id_,
-        TestChat.title,
-        TestChat.type_,
+        title=TestChat.title,
+        type=TestChat.type_,
```

Comment:
Any specific reason why this was changed? A failing test?

Reply:
because it's not in the same order:

`telegram._chat.Chat def __init__(self,
             id: int,
             type: str,
             title: Optional[str] = None,
...
`